nautobot_golden_config/datasources.py

```diff
@@ -36,6 +42,51 @@ def refresh_git_backup(repository_record, job_result, delete=False):  # pylint:
     )
 
 
+def refresh_git_gc_dynamic_remediations(repository_record, job_result, delete=False):  # pylint: disable=unused-argument
```

Comment:
I haven't looked at the code, but this part makes sense, I think that this is mostly what needs to be updated, more specifically there shouldn't be a reason to create a new model here.

Reply:
made some changes to remove the model and just load modules from the git repo when doing remediation; lmk what you think